2017-03-01-preview/Microsoft.Insights.ActionGroup.json

```diff
@@ -0,0 +1,172 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActionGroup.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "microsoft.insights",
```

Comment:
For RP namespaces we always keep the first letter capital. e.g. "Microsoft.Insights". You may need to fix in swagger also.